README.md

```diff
@@ -36,19 +82,36 @@ For description of other options
 crossref-verify --help
 ```
 
+## Configuring
+
+Configuration template with all options explained can be dumped with
+
+```sh
+crossref-verify dump-config
+```
+
+Currently supported options include:
+* Timeout for checking external references;
+* List of ignored folders.
+
+## For further work [↑](#crossref-verifier)
+
+- [ ] Support for non-Unix systems.
+- [ ] Support links detection in different languages, not only Markdown.
+  - [ ] Haskell haddock is first in turn.
+
 ## Issue tracker [↑](#crossref-verifier)
 
-We use [YouTrack](https://issues.serokell.io/issues/INT) as our issue
-tracker. You can login using your GitHub account to leave a comment or
-create a new issue.
+We use [GitHub issues](../../../issues) as our issues tracker.
```

Comment:
I think it's always called "issue tracker", not "issues tracker".

Reply:
Right.